testdata/zig/zig-hello-world/src/main.zig

```diff
@@ -0,0 +1,9 @@
+const std = @import("std");
+
+pub fn main() anyerror!void {
+    std.log.info("All your codebase are belong to us.", .{});
```

Comment:
Can we change this message. It's a bit grim 😄

Reply:
Its the default zig hello world - I think their sense of humour is  very Adams Family-esque, which I love :)

But yeah, I'll edit it to be less alarming.